src/python/pants/backend/jvm/targets/jvm_target.py

```diff
@@ -278,3 +288,7 @@ def excludes(self):
   @property
   def services(self):
     return self._services
+
+  @property
+  def is_thrift(self):
```

Comment:
I really don't like the `is_thrift` solution... it was deprecated for a reason, and should stay that way. And moreover, `JvmTarget` should not have awareness of other target types.